doc/src/properties/residue.toml

```diff
@@ -0,0 +1,72 @@
+[chainname]
+type = "string"
+
+PDB = """The **chainname** defines the name assigned to a protein chain by
+biologists. It is is a single character used to group residues which are linked
+together in a crystallographic file where it may not be unique."""
+
+mmCIF = """The **chainname** defines the name assigned to a protein chain by
+biologists. It is is a single character used to group residues which are linked
+together in a crystallographic file where it may *not* be unique. This name *is*
+unique to a given biological assembly, however."""
+
+MMTF = """The **chainname** defines the name assigned to a protein chain by
+biologists. It is is a single character used to group residues which are linked
+together in a crystallographic file where it may *not* be unique. This name *is*
```

Comment:
```suggestion
together in a crystallographic file where it may **not** be unique. This name **is**
```

Reply:
Is there a reason **bold** face is preferable to *italics*?

Reply:
Not really, I thought you wanted italics here =)